### DISCUSSION TITLE
Sample CSV

### DISCUSSION THREAD

**Discussion by mbaimuratov** [vmiklos/plees-tracker#326] [Q&A]
Hello! 
I am considering contributing to this project. Could you, please, provide a CSV file with sample data to be able to test exhaustively?

**Comment by vmiklos** [comment]
Here is a sample export one:

```
sid,start,stop,rating,comment
1,1665775289218,1665775498460,0,
2,1665775650542,1665775760633,0,
3,1665775762963,1665775778558,3,this is a note
```

and how it looks like on the UI:

![image](https://user-images.githubusercontent.com/13838/195927640-b0830629-aaa0-48d5-80a6-d8d3f9b480e9.png)

Is this enough?

What do you plan to contribute, btw? Fixing an issue, adding a feature?
